src/Components/RailSlider/Slider.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import Slick from "react-slick"
+
+import { Arrow } from "./Arrow"
+
+const SETTINGS = {
+  accessibility: true,
+  dots: false,
+  draggable: true,
+  infinite: false,
+  initialSlide: 0,
+  lazyLoad: false,
+  nextArrow: <Arrow direction='right' />,
+  prevArrow: <Arrow direction='left' />,
+  slidesToScroll: 1,
+  slidesToShow: 1,
+}
+
+interface Props {
+  artist: string,
+  initialSlide?: number
+}
+
+export const Slider = createFragmentContainer(
+  (props: Props) => {
```

Comment:
Hah, nice, never really considered inlining. Makes sense for such a short component 👍